system-tests/services/_scripts/external-volumes/teardown

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+pip install boto3
+pip install retrying
+
+cmd="./services/_scripts/external-volumes/volclean.py \
+-y -r $REGION -k $ACCESS_KEY_ID -s \"$SECRET_ACCESS_KEY\" \
+--ignore-metrics --tags 'Name:^integration-test-dcos-ui-$TEST_UUID'"
+
+MATCH_STRING="*Removing*Volumes*"
+TIMEOUT=30
+counter=0
+
+num_vols_delete=$NUM_VOLS_DELETE
+
+while [ "$num_vols_delete" -gt 0 ]
+do
+  echo "$num_vols_delete volume(s) left to delete"
+  while
+    result=$(eval "$cmd 2>&1")
+    echo $result
+    remove_string=$(echo $result | sed -n 's/^.*\(\Removing.*[0-9].*\Volumes\).*/\1/p')
+    num_removed=$(echo $remove_string | sed 's/[^0-9]*//g')
+    num_vols_delete=$(($num_vols_delete-${num_removed:-0}))
+    sleep 1s
+    counter=$(($counter+1))
+    if [[ "$counter" -ge $TIMEOUT ]]; then
+      echo "Timeout exceeded: External volume deletion"
+      exit 1
+    fi
+
+    [[ $result != $MATCH_STRING ]]
+  do
+    :
+  done
+done
```

Comment:
The purpose of `NUM_VOLS_DELETE` is to iteratively delete each volume. The volumes must be detached before the script can find the volume. It is done this way instead of having an arbitrary delay to wait for the volumes to detach.